examples/generalized_linear_models/GLM-truncated-censored-regression.ipynb

```diff
@@ -0,0 +1,1356 @@
+{
```

Comment:
<ul><li>Same comments as for first model</li><li>If I understood correctly, the <code>left_censored</code> and <code>right_censored</code> steps could also just be equalities, right? <code>left_censored = (y == bounds[0])</code>, since <code>y</code> can't, by definition, be below/above the bound</li></ul><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/30/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>

Reply:
<p>I thought it might be safer, numerically. But can use equality if preferred</p>

Reply:
Ah I see! No it's ok, but maybe just add this precision in a comment?

Reply:
<p>done</p>